packages/common/src/types.ts

```diff
@@ -0,0 +1,25 @@
+export type AwsCloudWatchEvent = {
```

Comment:
To state the obvious: the AWS SDK has no type for this?

Reply:
Not that I could find, no.